packages/flutter/lib/src/cassowary/solver.dart

```diff
@@ -158,13 +158,39 @@ class Solver {
   /// Attempts to add the constraints in the list to the solver. If it cannot
   /// add any for some reason, a cleanup is attempted so that either all
   /// constraints will be added or none.
+  ///
+  /// Check the [Result] returned to make sure the operation succeeded. Any
+  /// errors will be reported via the `message` property on the [Result].
+  ///
+  /// Possible [Result]s:
```

Comment:
here and elsewhere:
when mentioning the enum value, use a hyperlink. it won't work today unfortunately but eventually it will and then it'll be useful.